src/themes/mainTheme.ts

```diff
@@ -96,8 +96,12 @@ export default createMuiTheme({
       },
     },
     MuiTab: {
-      root: {
-        background: Colors.TabBackground,
+      wrapper: {
+        flexDirection: 'row-reverse',
+      },
+      labelIcon: {
+        minHeight: undefined,
+        paddingTop: undefined,
```

Comment:
hvorfor må du sette dette?

Reply:
MUI legger til litt ekstrastuff på padding og height hvis en Tab har icon (som er optional).
Tror det egentlig er ment til at enten har alle Tabs ikon, eller så har ingen, men har lagt inn slik at ikon bare vises for error tabs, og da legger plutselig MUI til en del ekstra plass i de tilfellene med minde det fjernes som her